packages/@react-types/shared/src/collections.d.ts

```diff
@@ -87,7 +87,7 @@ export interface Sortable {
 
 export interface SortDescriptor {
   /** The key of the column to sort by. */
-  column?: Key,
+  column: Key,
   /** The direction to sort by. */
   direction?: SortDirection
```

Comment:
oh I didn't realize that these were optional before. wonder why. should this one be required too?

Reply:
crud, i didn't realize that either
is requiring them too much of a breaking change?

Reply:
is it more correct than before? I feel like we might have been assuming it was non-null before in some places. So maybe bug fix?

Reply:
I'll try making the direction required and see if it highlights any problematic code

Reply:
I'll try making the direction required and see if it highlights any problematic code

Reply:
whelp, appears nothing broke that assumption. or whatever did is still not in strict mode yet